src/test/resources/regressions/features/defunc/waitgroup-simple1.gobra

```diff
@@ -26,6 +26,7 @@ func spawnThreads(n int) {
 	go worker(-1, wg, PredTrue!<!>) // spawns first worker
 	// { acc(wg.WaitGroupP(), 1/2) && acc(wg.WaitGroupStarted(), 1/2) && !wg.WaitMode() && wg.Token(PredTrue!<!>) }
 
+	invariant 0 <= i && i <= n
```

Comment:
Is this invariant now required?

Reply:
yes, on line 33 `i` occurs as a permission amount. This invariant is necessary such that `i` is not negative